modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/forms/input/InputLocalized.tsx

```diff
@@ -7,7 +7,8 @@ import ClayLocalizedInput from '@clayui/localized-input';
 import classNames from 'classnames';
 import React, {useEffect, useState} from 'react';
 
-import {FieldBase} from './FieldBase';
+import {LocalizedValue} from '../..';
```

Comment:
This looks like a weird import, what is it pointing to? Is there an `index` file at that location that it's implicitly importing from? I've never seen this pattern before.

Reply:
:grimacing: autocomplete